packages/config/tailwind-preset.js

```diff
@@ -106,6 +107,9 @@ module.exports = {
       boxShadow: {
         dropdown: "0px 2px 6px -1px rgba(0, 0, 0, 0.08)",
       },
+      borderWidth: {
+        "booker-width": "var(--cal-border-booker-width, 1px)",
```

Comment:
Named it booker-width so that it doesn't conflict with border-booker which is a color.